fun.kikoplay.KikoPlay.yml

```
@@ -0,0 +1,243 @@
+app-id: fun.kikoplay.KikoPlay
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: KikoPlay
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  # aria2c download
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-download
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  # for hardware acceleration
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --persist=.config/kikoplay
+
+add-extensions: 
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: 23.08
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+
+modules:
+  - shared-modules/luajit/luajit.json
+
+  - name: libass
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/pkgconfig
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.gz
+        sha256: d653be97198a0543c69111122173c41a99e0b91426f9e17f06a858982c2fb03d
+        x-checker-data:
+          type: anitya
+          project-id: 1560
+          stable-only: true
+          url-template: https://github.com/libass/libass/releases/download/$version/libass-$version.tar.gz
+
+  - name: libv4l2
+    cleanup:
+      - /sbin
+      - /bin
+      - /include
+      - /lib/*.la
+      - /lib*/*/*/*.la
```

```suggestion
      - "*.la"
```

you also probably need "*.a"